scheduler/data-flow/src/main/kotlin/io/seldon/dataflow/kafka/PipelineStatus.kt

```diff
@@ -104,4 +104,31 @@ open class PipelineStatus(val state: KafkaStreams.State?, var isError: Boolean)
 
     override var exception: Exception? = null
     override var message: String? = null
+
+    fun isActive(): Boolean {
+        return when {
+            this is StreamStarting || this is Started -> true
+            this.state in
```

Comment:
whats the difference between these two conditions?  is the second one happening at runtime perhaps?